tests/model_indexes/models.py

```diff
@@ -15,6 +15,14 @@ class Meta:
         ]
 
 
+class QuotedDbTable(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = '"t_quoted"'
+        indexes = [models.indexes.Index(fields=['name'])]
```

Comment:
It would be great to avoid creating a global model that will result in an extra table on every test suite execution since we are not going to query it. Could you define the model class in the test with `isolate_apps`?